ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -127,6 +127,7 @@ public int execute() {
       if (shouldLoadAtlasMetadata()) {
         addAtlasLoadTask();
       }
+      initiateRangerDenytask();
```

Comment:
you can do this only if deny config is enabled

Reply:
If config is disabled, it would make delete request if deny policy was created in previous iteration.

Reply:
If config is enabled, this task would add the ranger deny policy.
If config is disabled, this task would delete ranger deny policy if created previously.